pywinauto/application.py

```diff
@@ -1044,10 +1044,10 @@ def wait_cpu_usage_lower(self, threshold=2.5, timeout=None, usage_interval=None)
         if timeout is None:
             timeout = Timings.cpu_usage_wait_timeout
 
-        start_time = time.time()
+        start_time = time.clock()

```

Comment:
@vasily-v-ryabov, we may make these time.clock changes later or in a separate PR